flottform/server-standalone/src/routes/flottform/turn-control/global.ts

```diff
@@ -0,0 +1,9 @@
+let useTurnServer = false;
+
+export function getUseTurnServer() {
+	return useTurnServer;
+}
+
+export function setUseTurnServer(value: boolean) {
+	useTurnServer = value;
+}
```

Comment:
I wonder whether we can use this `$state` rune here, something like `export const useTurnServer = $state(false);`. But this should work already